### ISSUE TITLE
Update sample operator CR to include all possible config knobs for user reference

### ISSUE THREAD

**Issue by font** [gatekeeper/gatekeeper-operator#69]
This should also update the CSV as part of the bundle generation so that the ALM examples annotation includes a complete example for users to reference:
https://github.com/gatekeeper/gatekeeper-operator/blob/66712a15ddc73df06afd1c8f7c6163e2debe4143/bundle/manifests/gatekeeper-operator.clusterserviceversion.yaml#L5-L28

**Comment by ruromero** [comment]
I'd use the minimum required as the main example to get non-experienced users a simple way to get started.
As we can provide multiple examples, then we can append some other examples with a specific purpose so that UIs supporting it can allow you to pick one or use the default.
Maybe as an all-in-one example we can use this https://github.com/gatekeeper/gatekeeper-operator/blob/master/config/samples/gatekeeper_with_all_values.yaml

**Comment by font** [comment]
Yes, I agree a simple example makes sense. At the same time my concern is that the existing sample is used for documentation purposes within the CSV that is then placed onto operatorhub.io. I've seen some operators use a simple example like [etcd](https://operatorhub.io/operator/etcd) and others like [istio](https://operatorhub.io/operator/istio) use a more extensive example that may show all the config options possible. These examples are obtained from the `alm-examples` field of the CSV as shown in the snippet above.
